cegs_portal/search/templates/search/v1/search_results.html

```diff
@@ -294,11 +439,24 @@
 
         updateFacetState();
 
+        let searchFeatures = queryParams.getAll("feature_type");
+        featureCheckboxes.forEach(checkbox => {
+            checkbox.checked = searchFeatures.includes(checkbox.id);
+            checkbox.addEventListener("change", _ => {
+                updateFeatureState();
+                updateFeatureTypeTable();
```

Comment:
This shouldn't be here, it should be around line 505, wrapped in a `addCallback` like so:

```javascript
       state.addCallback(STATE_FEATURE_TYPES, (s, key) => {
             window.history.pushState({}, document.title, searchURL(s[key]));
             updateFeatureTypeTable();
         });
```

Reply:
I wrapped it previously and it set the facet parameter to feature_type and we receive an int() error. The table doesn't appear either unless you check a checkbox